include/maliput/geometry_base/road_geometry.h

```diff
@@ -162,6 +162,17 @@ class RoadGeometry : public api::RoadGeometry {
     strategy_ = std::make_unique<StrategyT>(this, std::forward<Args>(args)...);
   }
 
+  /// Returns the Geo Reference info of this RoadGeometry.
+  ///
+  /// The returned info will contain at most one of the projections described in
+  /// https://proj.org/en/stable/operations/projections/index.html under the "data" key.
+  /// Other key values may be present as well to complement the projection's values.
```

Comment:
No need to even mention PROJ4 format. That would be defined by the backend entirely.